airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/operation/DatabricksStorageOperation.kt

```diff
@@ -93,18 +92,9 @@ class DatabricksStorageOperation(
         )
     }
 
-    private fun prepareStagingTable(streamId: StreamId, destinationSyncMode: DestinationSyncMode) {
-        val rawSchema = streamId.rawNamespace
-        // TODO: Optimize by running SHOW SCHEMAS; rather than CREATE SCHEMA if not exists
-        destinationHandler.execute(sqlGenerator.createSchema(rawSchema))
```

Comment:
the `create schema` call now happens via destinationhandler.createNamespaces, so remove it from here